pkgs/development/compilers/gcc/6/default.nix

```diff
@@ -120,6 +120,9 @@ stdenv.mkDerivation ({
     repo = "gcc-vc4";
     rev = "e90ff43f9671c760cf0d1dd62f569a0fb9bf8918";
     sha256 = "0gxf66hwqk26h8f853sybphqa5ca0cva2kmrw5jsiv6139g0qnp8";
+  } else if stdenv.targetPlatform.isRedox then fetchGit {
```

Comment:
I think for the same reason we can't use fetchTarball in restricted eval, this will also have to be `fetchgit`